crawl-ref/source/lookup-help.cc

```diff
@@ -358,9 +358,11 @@ static vector<string> _get_god_keys()
     for (int i = GOD_NO_GOD + 1; i < NUM_GODS; i++)
     {
         god_type which_god = static_cast<god_type>(i);
+#if TAG_MAJOR_VERSION == 34
```

Comment:
should this function exist at all in TAG 35?

Reply:
Yes, definitely. It is used to power the `?/G` menu.